core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -238,7 +242,9 @@ describe('Enable correctness feedback and set correctness', function() {
 
     await explorationEditorMainTab.setInteraction('MultipleChoiceInput', [
       await forms.toRichText('Correct!'),
-      await forms.toRichText('Wrong!')
+      await forms.toRichText('Wrong!'),
+      await forms.toRichText('Not correct'),
+      await forms.toRichText('Wrong again')
     ]);
     await explorationEditorMainTab.addResponse(
       'MultipleChoiceInput', await forms.toRichText('Good!'),
```

Comment:
You can see more specifically where the test is failing by running this E2E test suite locally. But I'm guessing it may be because the lesson is invalid so it is failing at `workflow.publishExploration()`. Notice that we used to create a response for the item "Correct!" and also create a response for the item "Wrong!" However, we do not have a default response for the 2 new Multiple Choice choices you've added. You should do something similar to ~10 lines of code above where you go to the `default` response and set a `default feedback`.

Reply:
Yeah, I also thought of that but regarding the `default feedback` don't we have the `default feedback` for all the answers that are wrong and tells users to try again?
But yes this might be the reason for failing tests, I'll try it out and will let you know.
Thanks, @EricZLou.

Reply:
Hmmm i guess you're right. Can you try running it locally and watching where it fails?